core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -122,4 +160,34 @@
     height: 100%;
     width: 100%;
   }
+
+  @media only screen and (max-width: 600px) {
```

Comment:
Can we make the colors consistent with the "Translate to" dropdown in the Translate Text tab? Notably the highlight color is a medium gray there.

Reply:
Unfortunately, the 'select' HTML tag offers little to no customizability via CSS (the styles are mostly dependent on the user's browser -- from what I've read). So I've gone ahead and replaced it with a combination of divs stitched together to resemble a dropdown (similar to the 'Translate to' dropdown you mentioned).

I then applied almost all the styles from that dropdown (not just the background-color on highlight) to end up with this, could you please confirm if this design is ideal?
![image](https://user-images.githubusercontent.com/78612244/148233822-c9f12021-7cea-4258-9b17-c1eeff2a5d7c.png)

Reply:
LGTM, thanks!